docs/tutorials/helloworld.md

```diff
@@ -0,0 +1,308 @@
+A "Hello World" program is a computer program that outputs or displays the message "Hello, World!". Normally it is  the first program written by people learning to code. It is also used as a sanity test to make sure that a computer language is correctly installed and that the operator understands how to use it.
+
+This "Hello World" tutorial walks you through creating a project, installing Netcode for GameObjects (Netcode), and creating the basic components for your first networked game. It is also the base for the [Golden Path series](goldenpath_series/gp_intro.md).
+
+:::note
+The videos on this page were removed because they were out-of-date and caused more confusion than help. All videos in the Hello World and Golden Path series will be recreated and added back at a later time.
+:::
+
+## Create a new project in Unity
+
+1. Open the **Unity Hub**.
+1. Click **New**. 
+1. Select type ‘3D’
+1. Name the project *Hello World*.
+1. Select the location you want to save the project.
+
+## Install Netcode
+
+See the [install Netcode guide](../installation/installation.md) to install the Netcode package.
+
+## Create the Basic Components
+
+In this section we will create the basic building blocks of a multiplayer game.
+
+### Creating Network Manager and selecting the Transport
+
+In this section we will add a Network Manager and add Unity Transport (UTP) to our project.
+
+1. Right-click in the **Hierarchy** tab of the main Unity Window.
+1. Select **Create Empty**.
+1. Rename the `GameObject` **NetworkManager**.
+   
+  :::tip
+  We renamed the `GameObject` because:
+    * It makes it  easier to refer to later.
+    * There should only be one **NetworkManager**, this is the object that contains the  `NetworkManager` component. You may get unexpected results if you create more than one **NetworkManager**.
+  :::
+
+2. Select **NetworkManager**.
+3. Click **Add Component** in the Inspector Tab.
+4. Select **Netcode** from the list shown.
+5. Select `NetworkManager` Component from the list displayed.
+6. Inside the `NetworkManager` component tab, locate the  `NetworkTransport` field. 
+7. Click "Select Transport".
+8. Select `UnityTransport`.
+9. Save your scene.
+
+## Creating an object to spawn for each connected player
+
+This section adds in a player object and spawns it for each connected player.
+
+1. Right-click in the **Hierarchy** tab of the Unity Window to create a **3D Object > Capsule**
+1. Rename it **Player**. 
+1. While **Player** is selected, add a **Netcode** > `NetworkObject` component in the Inspector Tab.
+1. Click the **Assets** folder under the **Project** tab.
+2. Right-click inside the **Assets** folder to **Create** > **Folder** and call it **Prefabs**.
+3. Make **Player** a prefab by dragging it to **Prefabs** folder you just created.
+4. Delete **Player** from scene.
+
+  :::tip
+  We remove **Player**, because we will be using the network library to spawn the player. The library cannot track objects that start in the scene.
```

Comment:
I think this adjustment might help keep consistency:

We remove the **Player** object from the scene because we will be assigning this network prefab to the `Player Prefab` property in the `NetworkManager` component.  The library does not support defining a player object as an in-scene placed `NetworkObject`.